aws/modules/deploy/79-preferences-ingress.erb

```diff
@@ -3,6 +3,10 @@ kind: Ingress
 metadata:
   name: preferences-ingress
   namespace: gpii
+  <% if ENV["TF_VAR_cluster_name"].start_with?("dev") %>
```

Comment:
Can you use `ENV["TF_VAR_cluster_name"].start_with?("prd.", "stg.")` instead, for consistency with other code? 

(Really, this should be a function `is_long_lived?` or something but fow now we can keep it down to one variant of this logic.)

Reply:
Done